test/unit/offset.js

```diff
@@ -75,10 +75,11 @@ QUnit.test( "disconnected element", function( assert ) {
 	assert.equal( result.top, 0, "Retrieving offset on disconnected elements returns zeros (gh-2310)" );
 	assert.equal( result.left, 0, "Retrieving offset on disconnected elements returns zeros (gh-2310)" );
 	assert.equal( Object.keys( result ).length, 2, "Retrieving offset on disconnected elements returns offset object (gh-3167)" );
+	assert.equal( jQuery.isPlainObject( result ), true, "Retrieving offset on disconnected elements returns plain object (gh-3612)" );
```

Comment:
I guess I was just ensuring it wasn't a `ClientRect`. I could create a helper that checks both `isPlainObject` and ensures it is modifiable. WDYT?